components/light/binary.rst

```diff
@@ -20,6 +20,11 @@ The ``binary`` light platform creates a simple ON/OFF-only light from a
         name: "Desk Lamp"
         output: output_component1
 
+    output:
+      - id: light_output
```

Comment:
```suggestion
      - id: output_component1
```

Reply:
In that case, maybe better to update the name of the light config?